static/js/lib/site_content.ts

```diff
@@ -55,10 +54,29 @@ export function widgetExtraProps(field: ConfigField): Record<string, any> {
   }
 }
 
-const isMainContentField = (field: ConfigField) =>
+/**
+ * determine whether a ConfigField is a main content field or not.
+ * this means, basically, whether it's the main Markdown field for
+ * a site.
```

Comment:
```suggestion
 * a piece of site content.
```

Thanks for adding these docs!

Reply:
My editor is a bit smarter with typescript than it was with JS, so for functions that have docstrings they pop up when I'm writing code and it's very nice to have a brief little explanation! I'm trying to add it to most library code that I touch.